lib/Sema/CSBindings.cpp

```diff
@@ -1767,11 +1808,14 @@ void BindingSet::dump(llvm::raw_ostream &out, unsigned indent) const {
       auto *constraint = entry.second;
       PotentialBinding binding{constraint->getSecondType(),
                                AllowedBindingKind::Exact, constraint};
-      printBinding(binding);
```

Comment:
This looks unused now, as we seem to not need to create a potential binding in this scope and line bellow could be `auto defaultBinding = PrintableBinding::exact(constraint->getSecondType());`?